### PR TITLE
fix(core): WF| Able to enter negative value in 'n' interval of "Every time period" event

### LINKED ISSUE #67: WF| Able to enter negative value in 'n' interval of "Every time period" event
**Describe the bug**
WF| Able to enter negative value in 'n' interval of "Every time period" event

**To Reproduce**

**Expected behavior**
User should not be able to input negative values here. Also, allow integers only here. (no decimal values)

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by Deepika516**
you can not add neg values in workflows

GH-67

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #67 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/workflows-creator/src/lib/builder/group/group.component.ts [hunk 1]
```diff
@@ -590,6 +590,17 @@ export class GroupComponent<E> implements OnInit, AfterViewInit {
     }
   }
 
+  handleKeyPress(event:any) {
```
Comment (rohit-sourcefuse) [inline projects/workflows-creator/src/lib/builder/group/group.component.ts:593]: A small suggestion :
```javascript
handleKeyPress(event: any) {
  const keyCode = event.which || event.keyCode;
  const isDigit = keyCode >= 48 && keyCode <= 57;
  const isBackspaceOrDelete = [8, 46].includes(keyCode);
  const inputValue = event.target.value;
  const isValidInput = /^-?\d*\.?\d*$/.test(inputValue);

  if (!(isDigit || isBackspaceOrDelete) || !isValidInput) {
    event.preventDefault();
  }
}
```
we can combine the two event.preventDefault() calls into a single conditional statement.

Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/group/group.component.ts:593]: sure sir.

Comment (yeshamavani) [reply projects/workflows-creator/src/lib/builder/group/group.component.ts:593]: done ?? @Deepika516

Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/group/group.component.ts:593]: I  had a talk with @rohit-sourcefuse sir @yeshamavani he said no need to do this right now , its a suggestion.

Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/group/group.component.ts]: Done

**Pull request merged**